src/spans/awsSpan.js

```diff
@@ -67,7 +67,7 @@ export const getBasicSpan = () => {
   const account = awsAccountId;
 
   const readiness = isWarm() ? 'warm' : 'cold';
-  if (readiness === 'warm') {
+  if (readiness === 'cold') {
```

Comment:
Don't we need to `setCold` in this case?

Reply:
if the lambda is cold, the next time you want its to be warm (not opposite)